frontend/web/components/import-export/FeatureExport.tsx

```diff
@@ -0,0 +1,215 @@
+import { informationCircle } from 'ionicons/icons'
+import TagFilter from 'components/tags/TagFilter'
+import PanelSearch from 'components/PanelSearch'
+import FeatureListStore from 'common/stores/feature-list-store'
+import FeatureListProvider from 'common/providers/FeatureListProvider'
+import AppActions from 'common/dispatcher/app-actions'
+import FeatureRow from 'components/FeatureRow'
+import { FeatureState, ProjectFlag, TagStrategy } from 'common/types/responses'
+import ProjectStore from 'common/stores/project-store'
+import Utils from 'common/utils/utils'
+import Button from 'components/base/forms/Button'
+import {
+  useCreateFeatureExportMutation,
+  useGetFeatureExportsQuery,
+} from 'common/services/useFeatureExport'
+import InfoMessage from 'components/InfoMessage'
+import FeatureExportItem from './FeatureExportItem'
+
+type FeatureExportType = {
+  projectId: string
+}
+
+const FeatureExport: FC<FeatureExportType> = ({ projectId }) => {
+  const [environment, setEnvironment] = useState<string>(
+    ProjectStore.getEnvs()?.[0]?.api_key,
+  )
+  const [tags, setTags] = useState<(number | string)[]>([])
+  const [search, setSearch] = useState()
+  const [page, setPage] = useState(0)
+  const [tagStrategy, setTagStrategy] = useState<TagStrategy>('UNION')
+
+  useEffect(() => {
+    if (environment) {
+      AppActions.getFeatures(projectId, environment, true, null, null, page, {
+        search,
+        tag_strategy: tagStrategy,
+        tags: tags?.length ? tags : undefined,
+      })
+    }
+  }, [environment, tagStrategy, tags, search, projectId, page])
+  const [createFeatureExport, { isLoading: isCreating, isSuccess }] =
+    useCreateFeatureExportMutation({})
+
+  const { data: exports, refetch } = useGetFeatureExportsQuery({ projectId })
+  const onSubmit = () => {
+    const env = ProjectStore.getEnvironment(environment)
+    createFeatureExport({
+      environment_id: env.id,
+      tag_ids: tags,
+    }).then((res) => {
+      if (res?.data) {
+        toast('Your export is processing')
+      } else {
+        toast('Failed to export features')
+      }
+    })
+  }
+  const timerRef = useRef<NodeJS.Timer>()
+  useEffect(() => {
+    if (
+      exports?.results.find(
+        (featureExport) => featureExport.status === 'PROCESSING',
+      )
+    ) {
+      timerRef.current = setTimeout(() => {
+        refetch()
+      }, 200)
+    }
+  }, [exports, refetch])
+  return (
+    <div className='mt-4'>
+      <InfoMessage>
+        This will export the project's features using the default values from a
+        given environment.
+      </InfoMessage>
+      <div className='mb-2'>
+        <Tooltip
+          title={
+            <>
+              <strong>Environment</strong>
+              <IonIcon icon={informationCircle} />
+            </>
+          }
+        >
+          {
+            'Selecting an environment will determine the feature defaults used when importing to a project'
+          }
```

Comment:
Hmmm. Maybe this language could be tightened a bit to differentiate the environment people import to vs the other environments that take the defaults from the import.

Reply:
Yeah, I just picked up on this. I think @kyle-ssg is maybe confused here. The environment values are used only to overwrite the values in the environment you import to. The default values that exist on the feature in the project of the _exporting_ environment are the ones that are used in other environments in the import project. 

I would suggest that we _either_ change the wording as follows, or just remove this tooltip altogether. 

```suggestion
          {
            'Select the environment to use for the export.'
          }
```